content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -219,13 +49,13 @@ Teams:
   ....
 ```
 
-## 7. Configure continuous deployment to Netlify
+## 2. Configure Netlify
 
 You can now create your Netlify site and configure continuous deployments for it.
 
 ### Steps
 
-1. From the repository's folder, run `netlify init`.
+1. From the repository root, run `netlify init`.
```

Comment:
```suggestion
1. From your project folder, run `netlify init`.
```
Maybe?

Reply:
How about we meet in the middle? 
```suggestion
1. From your project root, run `netlify init`.
```

Reply:
I dislike `root`, as I don't know what it means as a windows user and am confused :D 

But yeah, no hard feelings.